code_snippets/java/README.md

```diff
@@ -4,7 +4,7 @@ Sample project configuration of a Restate service using the Java interface and H
 
 * [`build.gradle.kts`](build.gradle.kts)
 * [Service interface definition `greeter.proto`](src/main/proto/greeter.proto)
-* [Service class implementation `Greeter`](src/main/java/Greeter.java)
```

Comment:
Is this README.md used for anything in our documentation? I am asking because I am wondering why the lower sections about downloading the example etc. are needed.

Reply:
I will remove the README indeed